app/controllers/replies_controller.rb

```diff
@@ -139,10 +157,10 @@ def create
           flash.now[:error] = "This looks like a duplicate. Did you attempt to post this twice? Please resubmit if this was intentional."
           @allow_dupe = true
           if most_recent_unseen_reply.nil? || (most_recent_unseen_reply.id == last_by_user.id && @unseen_replies.count == 1)
-            preview(reply)
+            preview_reply(reply)
```

Comment:
Why the rename?

Reply:
'Cause I split the original "preview" function into a part that is shared between "literally previewing" and "editing multi replies" and a part that is specific to each of those, so now `preview_reply` does the "literally previewing"-specific logic then calls `preview`, while `add_to_multi_reply` does the "editing multi replies"-specific logic then calls `preview`

Reply:
Ah, that makes sense.